package.json

```diff
@@ -44,6 +44,7 @@
     "cookie-parser": "1.3.2",
     "copy-webpack-plugin": "3.0.1",
     "creditcards": "2.1.2",
+    "cross-env": "4.0.0-beta.0",
```

Comment:
@blowery, @gwwar  - I bet it's still dev if it doesn't get into server or client bundles, right?